index.html

```diff
@@ -5,26 +5,28 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Portfolio</title>
   <script src="https://kit.fontawesome.com/39e3f91239.js" crossorigin="anonymous"></script>
-  <link rel="stylesheet" href="desktopVersion.css">
-  <link rel="stylesheet" href="small_version.css">
+  <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
     <header>
-    <nav>
-      <a class="logo"href="#homa">Emmabgra</a>
-      <img class="menu" src="img/Union.svg" alt="">
-      <ul class="navbar__ul show__links">
-        <li class="navrbar__link"><a href="#portfolio">Portfolio</a></li>
-        <li class="navrbar__link"><a href="#about">About</a></li>
-        <li class="navrbar__link"><a href="#contact">Contact</a></li>
+      <nav class="nav">
+        <a class="logo"href="#homa">Emmabgra</a>
+        <img class="menu" src="img/Union.svg" alt="">
+       <ul class="navbar__ul" id="navbar__ul">
+        <i class="fa-solid fa-xmark"></i>
+        <li><a href="#contact" class="navMenu">Contact</a></li>
+        <li><a href="#portfolio" class="navMenu">Portfolio</a></li>
+        <li><a href="#about" class="navMenu">About</a></li>
```

Comment:
- [ ] As the previous Code Reviewer was suggesting 😄 , there's should not be differences between your design and the Figma design 💯 . When the mobile menu is enable there should be some `blur` on the main page, It will increase the esthetic of your page and even the user experience 😿 . Kindly improve it @sibomana123 

Figma design
![257200058-edf57a0d-fcd4-4fe6-a5f2-77f9d6c6b5ff](https://github.com/sibomana123/MV__Portfolio/assets/80458199/2404bd7e-85b2-4e52-bd07-c356660444d0)

Your design

![Capture](https://github.com/sibomana123/MV__Portfolio/assets/80458199/d79c3b66-f334-4f9c-bf00-9e12bd1e5762)